src/main/java/com/google/javascript/gents/CollectModuleMetadata.java

```diff
@@ -180,6 +191,10 @@ void registerProvidesModule(Node n, String file, String namespace) {
     /** {@code True}, if the module has any imports (e.g.{@code goog.require}). */
     private boolean hasImports = false;
 
+    private String namespace = "";
```

Comment:
Add a jsdoc for this new field. The word 'namespace' is used through out the code base to mean slightly different things, that create a lot of confusion. If you can quantify ('originalNamespace'?) it somehow it would be even better.

Reply:
Done. Renamed to `namespaceInJS`.